Capri.Services/PromoterDeleter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Capri.Database;
+using Capri.Database.Entities;
+using Capri.Database.Entities.Identity;
+using Capri.Web.ViewModels.Promoter;
+
+namespace Capri.Services
+{
+    public class PromoterDeleter : IPromoterDeleter
+    {
+        private readonly ISqlDbContext _context;
+
+        public PromoterDeleter(ISqlDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<IServiceResult<Promoter>> Delete(Guid id)
+        {
+            var promoter = 
+                await _context
+                .Promoters
+                .FirstOrDefaultAsync(_ => _.Id == id);
```

Comment:
I think lower dash is mostly used when you want to omit some argument. I would rather use some letter, like `x`, `n`. In such case event clearer will be to use `p` as `promoter`.